lib/cache/adapters/memory_adapters/queue_adapter.rb

```diff
@@ -7,18 +7,27 @@ class QueueAdapter
           def initialize(size)
             @size = size
```

Comment:
@maksimf can we make a tiny name refactor here? I would name this: @max_size and rename @size_counter to @current_size ?